app/head.mjs

```diff
@@ -0,0 +1,19 @@
+import { getStyles }  from '@enhance/arc-plugin-styles'
+
+export default function Head () {
+  const styles = process.env.ARC_LOCAL
```

Comment:
Remove as it may be confusing / annoying to users.
Just use the `linkTag` and we can add how to use inline `styleTag` in the docs.

Don't worry about the fallback `head.mjs` we will leave it as is where it is.